pkgs/tools/misc/pridefetch/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, lib, python39, zip }: let
+  version = "1.0.0";
+  sha256 = "sha256-/o4er8bO/3HUFXzP+sC+5DYv9EwmxW05o1RT5fEulEg=";
+
+  name = "pridefetch";
+  src = fetchFromGitHub {
+    owner = "SpyHoodle";
+    repo = name;
+    rev = "v" + version;
+    inherit sha256;
+  };
+in stdenv.mkDerivation {
+  inherit name version;
+  buildInputs = [
+    (python39.withPackages (pythonPackages: with pythonPackages; [
+      distro
+    ]))
+    zip
+  ];
+  unpackPhase = "true";
+  installPhase = ''
+    mkdir -p $out/bin
+    cd ${src}/src
+    zip -r $out/pridefetch.zip *
+    echo '#!/usr/bin/env python' | cat - $out/pridefetch.zip > $out/bin/pridefetch
+    chmod +x $out/bin/pridefetch
+    rm $out/pridefetch.zip
+  '';
+  meta = with lib; {
+    description = "Print out system statistics with pride flags";
+    longDescription = ''
+      Pridefetch prints your system statistics (similarly to neofetch, screenfetch or pfetch) along with a pride flag.
+      The flag which is printed is configurable, as well as the width of the output.
+    '';
+    homepage = https://github.com/SpyHoodle/pridefetch;
+    license = "bsd";
```

Comment:
We have licenses that can be referred to in the lib, please use those instead of strings.